index.html

```diff
@@ -298,7 +302,6 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
 							<img class="col-sm-2" alt="Abishek V Ashok" src="img/abishek.jpg">
 							<img class="col-sm-2" alt="Mochammad Rafi Al Rizkan" src="img/rafi.png">
 							<img class="col-sm-2" alt="Mehmet Çağrı Arı" src="img/cagriari.jpg">
-							<img class="col-sm-2" alt="Reza Juliandri" src="img/reza_juliandri.jpg">
 					</div>     
```

Comment:
Hi @STEdirisuriya ! Why have you removed this line? :)

Reply:
Nothing It isn't important